test/libs/metrics/top_level_metrics_test.py

```diff
@@ -41,11 +43,22 @@
 ]
 
 
+class TimeseriesLit(UserList):
```

Comment:
TimseriesLit was pretty confusing to me until I read the comment, maybe `TimeseriesLiteral`?